ddp-tool.yaml

```
@@ -0,0 +1,40 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: ddp-tool
+  version: 1.0.34.0_git20241028
+  epoch: 0
+  description: Intel
+  copyright:
+    - license: Apache-2.0
```

Seems like BSD License
```suggestion
    - license: Apache-2.0
```